### DISCUSSION TITLE
Alert - Discontinuing pages / features

### DISCUSSION THREAD

**Discussion by rafam4rtins** [twilio-labs/paste#2824] [General] [answered]
Hi folks, I'm working on a possible pattern when discontinuing pages & features, and for this I'm using the alert component. Paste does suggest copy should be brief and contain just high priority information, and in simple cases it works really well, but in some possible scenarios it might need larger copy and multiple links.

In the "don’ts" of the pattern I'll suggest a character limitation number, but I'd like to know if there is some guidance on how to use alerts with different information sizes and potentially breaking lines or adding bullet points to it, similar to what we have on Callouts. 

Find a screenshot with some examples.
Thank you!

![Discontinue alerts format](https://user-images.githubusercontent.com/104998142/201739887-459e545d-89d4-48ab-8fd2-87aaefe2b8ce.png)

**Comment by SiTaggart** [comment, marked as answer]
Hi @rafam4rtins 

I think generally in this case, the guidelines are doing their job. We really only want alerts to be used for very important system level information, and as such they should be written following our content guidelines, brief and contain high priority information. Examples include:

- API is down
- Account over-due
- Account locked
- Platform maintenance windows
- Active incidents

Because your content doesn't really fit those scenarios, and you correctly identify the callout as already accommodating your needs, I'm wondering what led you to an alert? Would a callout suit your need here?

**Reply by rafam4rtins** [reply]
Thanks for writing back @SiTaggart

Yep, I did some tests using Callout, but I have focused on the Alert because it seemed more appropriate for the application, as the alert description says "high-priority or time-sensitive information". I will get back to the Callouts, it should work well 🙌 Thanks!
